core/utils.go

```diff
@@ -301,3 +303,37 @@ func FileExists(path string) (bool, error) {
 
 	return false, err
 }
+
+// get body text with a text editor
+// works with terminal based editors
```

Comment:
We should also be able to support gui editors such as vscode. I wonder how bubble tea does it.

Reply:
`code --wait --new-window` these flags has to be used otherwise vscode won't block execution.
I tried the bubbletea too, it doesn't work without '--wait' flag
and there is a linting err here. It asks to validate the input code name.

Reply:
Ok using `--wait` is probably fine, it is probably expected from other cli tools like git anyway.

Reply:
Should there be a list of supported editors or anything like that so that program execution is predictable?.
And this way the linting err will also go away. It's about not validating input(editor name) before executing it.

Reply:
If you are talking about the gsec:G204 error you can add a nolint rule to the line above for it. In theory you could do some stuff maybe, but its a local running program and we are just launching your EDITOR you assigned for us, so really its on you.

I don't know about a list of editors, maybe explain in the docs you will need to add whatever flag to wait for the program to exit. There is probably already one out there since we are mimicking git behavior with there editor option.

Reply:
Will this be enough for the doc?
```go
	cmd.Flags().BoolVarP(
		&n.BodyEditor,
		"editor", "e",
		false,
		`Edit body using 'EDITOR' env variable (default: "vim"). For VSCode: 'EDIROR="code --wait --new-window"'`,
	)
```

Reply:
I wouldn't mention the VSCode specific config, only that we default to vim. Although, I am still on the fence about defaulting to vim versus returning an error if `--editor` is used but `EDITOR` var is empty.